.copr/make-srpm.sh

```diff
@@ -1,24 +1,20 @@
 #!/bin/bash
 
-set -ex
+set -eux
 
 outdir="$1"; shift
 
-dirname="$(dirname "$0")"
+rootdir="$(realpath -m "$0"/../..)"
```

Comment:
This is just a style preference, but could you move the rest of the path under the quotes please? (`"$0/../.."`)